### PR TITLE
Reorganise Quickstart section with new branding + move LUKSO wallet up on Network pages

### PR DESCRIPTION
**Pull request by CJ42**
Screenshot 2024-11-01 at 08 12 06

Screenshot 2024-11-01 at 08 12 27

Screenshot 2024-11-01 at 08 12 17

### CODE REVIEW

**Review by richtera — APPROVED** [review]

**Comment by CJ42** [comment]
Converting to draft for now

## Remaining tasks

- [ ] Remove new images or replace with new branded assets
- [ ] Rename pages according to new titles + setup redirects
- [ ] Remove duplicate content for Benefits of LSPs + merge both content
- [ ] Remove setup instructions related to LUKSO Playground + move them in the playground repository

**Review by JeneaVranceanu — APPROVED** [review]

File: docs/learn/benefits-lukso-standards.md [hunk 1]
```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
...
+| **📢 Notification and customized reactivity**  | The 🆙 can be customized to react differently based on different events it receives. For instance, the default implementation automatically register new received assets.                                                                                                                                                          |
+| **⚙️ Extendability with pluggable Extensions** | New features (like new function selectors not present by default in the smart contract code) can be added to a Universal Profile, using a system of extensions. <br/><br/>See our guide [**Extending Universal Profile functionalities**](./universal-profile/advanced-guides/extend-profile-functionalities.md) for more details. |
+
+### 🗄️ Metadata in one place
+
+With traditional web3 wallets, a user has to fill the same infos every time it registers on a new dApp (_e.g: username, biography, social media accounts_). A Universal Profile stores user's data in one single place: the contract's storage. any dApp can then retrieve the same information from the same place, making dApp onboarding easier and faster.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:40]:
```suggestion
With traditional web3 wallets, a user has to fill the same infos every time it registers on a new dApp (_e.g: username, biography, social media accounts_). A Universal Profile stores user's data in one single place: the contract's storage. Any dApp can then retrieve the same information from the same place, making dApp onboarding easier and faster.
```

File: docs/learn/benefits-lukso-standards.md [hunk 2]
```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
...
+| Feature                                        | Benefits                                                                                                                                                                                                                                                                                                                           |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **🗄️ Metadata in one place**                   | Finished re-filling the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]: Hmm, "finished re-filling" sounds a little bit strange. `No more re-filling` sounds better but with that wording we will have 2 sentences starting identically. 
```suggestion
| **🗄️ Metadata in one place**                   | Removes the requirement to refill the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
```

File: docs/learn/benefits-lukso-standards.md [hunk 3]
```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
...
+| **🎨 Customizable metadata**                   | Make your Universal Profile stand out from others! Give it a profile and cover image, and add as many amount of information with its unlimited storage.                                                                                                                                                                            |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]: Is the storage really unlimited? 🤔

File: docs/learn/benefits-lukso-standards.md [hunk 4]
```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
...
+| **⛽️ Gas-Less transactions**                  | No more need to buy and hold native tokens to pay for the gas. Use the transaction relayer plugged to the Universal Profile from the start to get started.                                                                                                                                                                         |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]: "transaction relayer plugged to the Universal Profile from the start" - should we specify that it's only related to profiles deployed using universalprofile.cloud or at least that's what we provide?

```suggestion
| **⛽️ Gas-Less transactions**                  | No more need to buy and hold native tokens to pay for the gas. Use the transaction relayer plugged to the Universal Profile from the start to get you going.                                                                                                                                                                         |
```

File: docs/learn/benefits-lukso-standards.md [hunk 5]
```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
...
+| **🔑 Multi-Control through permissions**       | Universal Profiles can be controlled by multiple EOAs with various permission levels, held across different devices or representing dApps. <br/>Each can have specific access rights (token transfers, playlist management, or account recovery).                                                                                  |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]:
```suggestion
| **🔑 Multi-Control through permissions**       | Universal Profiles can be controlled by multiple EOAs with various permission levels, held across different devices or representing dApps. <br/>Each can have specific access rights (token transfers, playlist management, account recovery, etc.).                                                                                  |
```

File: docs/learn/benefits-lukso-standards.md [hunk 6]
```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
...
+<br /><br />
+</div>
+
+### ⛽️ Gas-Less transactions
+
+:::info Opting out of the relayer
+
+**Using the relayer and always gas less transactions is not mandatory!**
+
+Users can also choose to not rely on the [Transaction Relay Service](../standards/accounts/lsp15-transaction-relayer-api.md) and the pay the gas fees of the transaction themselves by funding the [controller](./concepts.md#controller) of their Universal Profile with native LYX.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:81]:
```suggestion
Users can also choose not to rely on the [Transaction Relay Service](../standards/accounts/lsp15-transaction-relayer-api.md) and pay the gas fees of a transaction themselves by funding the [controller](./concepts.md#controller) of their Universal Profile with native currency (e.g. LYX).
```

File: docs/learn/benefits-lukso-standards.md [hunk 7]
```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
...
+2. creating a new address
+3. registering to a crypto exchange or a service to buy crypto
+4. performing KYC
+5. buying native crypto by card or bank transfer
+6. _in some cases,_ requesting the bank to authorize the transaction to buy crypto (some user banks restrict from buying crypto online)
+7. finally start using dApps!
+
+This improves onboarding experiences in the following ways:
+
+- most new web3 users does not get their head around the concept of "gas" and having to pay fees every time they interact with a dApp (compared to interacting with other applications on the internet).
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:97]:
```suggestion
- most of the new web3 users do not get their heads around the concept of "gas" and having to pay fees every time they interact with a dApp (compared to interacting with other applications on the internet).
```

File: docs/learn/benefits-lukso-standards.md [hunk 8]
```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
...
+- native and experienced web3 users need to acquire more native currency from various chains just to _"play around with dApps"_ and explore, increasing the costs even more for users present across many chains.
+- users can also bridge native currency from one chain to another (_e.g: having to bridge ethers from Ethereum to Base to use dApps on Base_), but this require giving up some funds on one chain as a result.
+- developers looking to build dApps on new chains neex to acquire test LYX from testnet faucets, but most of them now require to hold a minimum amount of native currency on the main network to avoid spam and over-usage of the faucet.
+
+As Universal Profile support gas-less transactions, it is possible to add multiple relayers and switch between their preferred one at the time of the transaction.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:102]:
```suggestion
As Universal Profile supports gas-less transactions, it is possible to add multiple relayers and switch between them before the confirmation of the transaction.
```

File: docs/learn/benefits-lukso-standards.md [hunk 9]
```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
...
+4. performing KYC
+5. buying native crypto by card or bank transfer
+6. _in some cases,_ requesting the bank to authorize the transaction to buy crypto (some user banks restrict from buying crypto online)
+7. finally start using dApps!
+
+This improves onboarding experiences in the following ways:
+
+- most new web3 users does not get their head around the concept of "gas" and having to pay fees every time they interact with a dApp (compared to interacting with other applications on the internet).
+- native and experienced web3 users need to acquire more native currency from various chains just to _"play around with dApps"_ and explore, increasing the costs even more for users present across many chains.
+- users can also bridge native currency from one chain to another (_e.g: having to bridge ethers from Ethereum to Base to use dApps on Base_), but this require giving up some funds on one chain as a result.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:99]:
```suggestion
- users can also bridge native currency from one chain to another (_e.g: having to bridge ethers from Ethereum to Base to use dApps on Base_), but this requires giving up some funds on one chain as a result.
```

File: docs/learn/benefits-lukso-standards.md [hunk 10]
```diff
@@ -47,18 +50,62 @@ As shown from the table above, the [Universal Profile Browser Extension](/insta
...
+- developers looking to build dApps on new chains neex to acquire test LYX from testnet faucets, but most of them now require to hold a minimum amount of native currency on the main network to avoid spam and over-usage of the faucet.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:100]: Suggesting to have similar terms `mainnet` and `testnet`.

```suggestion
- developers looking to build dApps on new chains need to acquire test LYX from testnet faucets, but most of them now require to hold a minimum amount of native currency on the mainnet to avoid spam and over-usage of the faucet.
```

File: docs/learn/benefits-lukso-standards.md [hunk 11]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+| **🔘 Easier functions for developers**                                | You are a developer? Stop learning different functions for each new ERC token standard! (ERC20, ERC721, ERC1155, etc...).<br/>LSP7 and LSP8 use the **same function names** for transfer and operator approvals, whether it is a token or NFT transfer.                                                                             |
+| **🗄️ Unlimited & Dynamic Metadata for Tokens & NFTs**                 | Your NFT is not only an image. It can now hold as many information as imagined (custom traits, attributes, ...). This information can evolve overtime.                                                                                                                                                                              |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]:
```suggestion
| **🗄️ Unlimited & Dynamic Metadata for Tokens & NFTs**                 | Your NFT is not only an image. It can now hold as many information as can be imagined (custom traits, attributes, ...). This information can evolve overtime.                                                                                                                                                                              |
```

File: docs/learn/benefits-lukso-standards.md [hunk 12]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+| **䷓ Flexible Batching functionalities**                               | Distibute multiple tokens or NFTs easily to users with `transferBatch(...)` or perform multiple actions in a single transaction with `batchCalls(...)`. <br/> _(e.g: authorize multiple operators, transfer an NFT and update its metadata)_.                                                                                       |
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md]:
```suggestion
| **䷓ Flexible Batching functionalities**                               | Distribute multiple tokens or NFTs easily to users with `transferBatch(...)` or perform multiple actions in a single transaction with `batchCalls(...)`. <br/> _(e.g: authorize multiple operators, transfer an NFT and update its metadata)_.                                                                                       |
```

File: docs/learn/benefits-lukso-standards.md [hunk 13]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+    src={require('/img/learn/token-nft-metadata-examples.png').default}
+    alt="`Examples of metadata for different NFTs."
+    width="1200"
+  />
+<br/>
+<i>3 x different NFT collections with their unique traits and characteristics.</i>
+<br /><br />
+</div>
+
+Like Universal Profile, digital assets based on the LSP7 and LSP8 standards can hold an **unlimited quantity of metadata** for any various type of information considered relevant to the token or NFT collection. This can include:
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:163]: `any various type` - it feels like it should be either `any type` or `various type`. Reading `any various` sounds tautological, at least to me 🙂 

```suggestion
Like Universal Profile, digital assets based on the LSP7 and LSP8 standards can hold an **unlimited quantity of metadata** for any various type of information considered relevant to the token or NFT collection. This can include:
```

File: docs/learn/benefits-lukso-standards.md [hunk 14]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+
+- Icons and image.
+- The creators of the tokens / NFT collection.
+- The list of exchanges where this token can be found.
+
+LSP8 also allows to create more complex and rich NFT collections where **each NFT can have its own custom metadata**. Setting metadata specific to each NFT is done using the standardized functions [`setDataForTokenId(...)` and `setDataBatchForTokenIds(...)`](../contracts/overview/NFT/set-nft-metadata.md). This is useful for instance for:
+
+- NFTs that need to hold specific items like digital clothing items for avatars, or weapons / body armors in a video game.
+- dynamic NFTs, where any information, traits or attributes of a NFT could be programmed to change or evolve overtime according to certain conditions, logics or triggers.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:172]:
```suggestion
- dynamic NFTs, where any information, traits or attributes of an NFT could be programmed to change or evolve overtime according to certain conditions, logics or triggers.
```

File: docs/learn/benefits-lukso-standards.md [hunk 15]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+
+### 📢 Notify on transfers
+
+On each token transfers, bother sender & recipient are notified that a token transfer happened. This notification includes a specific notification ID (_e.g: "I have sent / received some tokens / NFTs"_. See typeIds for more infos).
+
+In addition, it is possible to program and _"plug-in"_ custom logic contracts called [**Universal Receiver Delegate**](../standards/accounts/lsp1-universal-receiver-delegate.md) to the sender and recipient, so that they can automatically react based on these different type of notifications. For example:
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:178]:
```suggestion
In addition, it is possible to program and _"plug-in"_ custom logic contracts called [**Universal Receiver Delegate**](../standards/accounts/lsp1-universal-receiver-delegate.md) to the sender and recipient, so that they can automatically react based on these different types of notifications. For example:
```

File: docs/learn/benefits-lukso-standards.md [hunk 16]
```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
...
+
+- Registering / Removing tokens from your list of received assets.
+- Block receiving spam tokens that can contain malicious logic to steal user's funds if interacting with such token contracts, by filtering them against a spam or blacklist registry.
```
Comment (JeneaVranceanu) [inline docs/learn/benefits-lukso-standards.md:181]: I suggest `Block incoming spam` or `Prevent receiving spam`

```suggestion
- Block incoming spam tokens that can contain malicious logic to steal user's funds if interacting with such token contracts, by filtering them against a spam or blacklist registry.
```

**Review by JeneaVranceanu — CHANGES_REQUESTED** [review]

**Comment by mustafademiray** [comment]
some of the images in the boxes are not loading 
image

File: docs/learn/benefits-lukso-standards.md (hunk 3 above)
Comment (CJ42) [reply docs/learn/benefits-lukso-standards.md]: > Is the storage really unlimited? 🤔

Yes it is practically unlimited because the data keys are `bytes32` which makes a very big range `2e256`

**Comment by CJ42** [comment]
> some of the images in the boxes are not loading image

@mustafademiray It is intended, as we do not have branded pictures for these. Suggest we add them later :)

File: docs/learn/benefits-lukso-standards.md (hunk 3 above)
Comment (JeneaVranceanu) [reply docs/learn/benefits-lukso-standards.md]: Ah, okay. I was thinking rather about the `bytes` part for the values. How much can we upload and store? I guess it's limited by the 1) block's gas limit of 42m and 2) physical storage.

Comment (CJ42) [reply docs/learn/benefits-lukso-standards.md]: Good question, I am actually not sure. A burnt pix has a very large amount of metadata in its `LSP4Metadata`, but we haven't tested the boundaries yet for a single `setData(bytes32,bytes)` transaction. 
Yes technically, it is limited to the block's gas limit for setting.

See the value under the `LSP4Metadata` here :)
https://erc725-inspect.lukso.tech/inspector?network=mainnet&address=0x3983151e0442906000dab83c8b1cf3f2d2535f82

Comment (JeneaVranceanu) [reply docs/learn/benefits-lukso-standards.md]: Thanks!
